sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/UnusedPrivateMember.cs

```diff
@@ -75,6 +75,17 @@ private struct Struct { }// Noncompliant {{Remove the unused private type 'Struc
         private delegate void Delegate2(); // Noncompliant {{Remove the unused private type 'Delegate2'.}}
         private event Delegate Event; //Noncompliant {{Remove the unused private event 'Event'.}}
         private event Delegate MyEvent; //Noncompliant {{Remove this unread private field 'MyEvent' or refactor the code to use its value.}}
+        private int[][] array = new int[0][];
+        private Dictionary<int, int> used = new Dictionary<int, int>();
+        private Dictionary<int, int> unused = new Dictionary<int, int>(); // Noncompliant
+
+        public int GetValue(int x, int y) => array[x][y];
+
+        public int GetItem(int i) => used[i];
+
+        private Dictionary<int, int> GetDictionary() => used;
```

Comment:
Can you return null here instead? Like this, we're not sure if `used` is used due to `used[i]` or this call. Having `used[i]` has more value as originally intended.

```suggestion
        private Dictionary<int, int> GetDictionary() => null;
```

Reply:
In this case `.Fixed.cs` and `.Fixed.Batch.cs` need to be updated too.